common/openapi.yml

```diff
@@ -264,6 +264,8 @@ components:
         last_modified_timestamp:
           type: string
           format: date-time
+        error_messages:
+          type: string
```

Comment:
Since we're returning nulls, you might want/need to mark this as either `optional: true` or `required: false` (I'm not sure which!)... or optional might be the default if we don't specify anything?

Reply:
It didn't like/recognize `optional: true` so I opted for `required: false`